### ISSUE TITLE
rootkit

### ISSUE THREAD

**Issue by DARK-DEVIL-66** [UnamSanctam/SilentCryptoMiner#2015]
i have qt
ex: i add the biginig of the name Dialerpayload.exe, i run the your services64 exe & Dialerpayload.exe  in victem pc from taskshedule every logon
so now what happen in next restart actually my doubt is exe was hidden in particuler location i can't execute Dialerpayload  Exe via cmd after rootkit execution

it 's persist ? or if you have ideaHOW TO MAKE MY PAYLOAD HIDDEN AND PERSISTANCE WITH TASK SHEDULER PLS SUGGEST ME

**Comment by DARK-DEVIL-66** [comment]
> No it's not persistent, but you might have run the Services64.exe before the dialerpayload.exe.

you mean i can execute service64 first 2nd Dialerpay;load.exe ?


wich is the root kit working pettern ? or both are ok


01st rootkit----->>2nd payload
01st payload---->>02nd rootkit

**Comment by UnamSanctam** [comment]
The rootkit will hide the Dialerpayload.exe file so you should run the Dialerpayload.exe file before you run the rootkit.

**Comment by DARK-DEVIL-66** [comment]
{
"access-log-file": null,
"access-password": null,
"algo-ext": true,
"api": {
"id": null,
"worker-id": null
},
"http": {
"enabled": false,
"host": "127.0.0.1",
"port": 0,
"access-token": null,
"restricted": true
},
"background": false,
"bind": [
{
"host": "0.0.0.0",
"port": 8888,
"tls": true
},
{
"host": "::",
"port": 8888,
"tls": true
}
],
"colors": true,
"title": true,
"custom-diff": 120000,
"custom-diff-stats": true,
"donate-level": 0,
"log-file": null,
"mode": "nicehash",
"pools": [
{
"algo": "rx/0",
"url": "sg.zephyr.herominers.com:1123",
"user": "ZEPHYR2s5bnTUHNSTKfG9Gc6eC88LFNpEcHjChP2HTi1Wt1pry5XXqBWrLeUUYzzurC22ySZHLwvRgmSAvNcDXsZKm3K9uKBVD33w=120000",
"pass": "Saba-zephyr",
"keepalive": true,
"enabled": true,
"tls": true
}
],
"retries": 2,
"retry-pause": 1,
"reuse-timeout": 0,
"tls": {
"enabled": true,
"protocols": null,
"cert": null,
"cert_key": null,
"ciphers": null,
"ciphersuites": null,
"dhparam": null
},
"user-agent": null,
"syslog": false,
"verbose": false,
"watch": true,
"workers": true
}

i want to set custom diff 120000
so it's my config anything wrong plase tell me
Thank's

**Comment by DARK-DEVIL-66** [comment]
[2023-12-08 08:58:05.323] net 0000 new job from sg.zephyr.herominers.com:1123 diff 480045 algo rx/0 height 137091

xm-proxy console showing fallowing diff

[2023-12-08 08:58:05.323] net 0000 new job from sg.zephyr.herominers.com:1123 diff 480045 algo rx/0 height 137091

**Comment by DARK-DEVIL-66** [comment]
xmrig.exe -o serverip:8888 --tls -B --background --cpu-max-threads-hint=20

it's command used from my mining rig !
(--cpu-max-threads-hint=20) ignoring the argument why ?

**Comment by DARK-DEVIL-66** [comment]
![image](https://github.com/UnamSanctam/SilentCryptoMiner/assets/149578298/f1540f88-3879-4358-b0d2-4d0ebc2216f2)


![image](https://github.com/UnamSanctam/SilentCryptoMiner/assets/149578298/0d6367a0-35c0-469f-9b59-fc26951c900e)


what is the differance

**Comment by DARK-DEVIL-66** [comment]
gcc vs msv differant anything special ?

**Comment by UnamSanctam** [comment]
Not really, it's just the compiler. You can get the MSVC version from the xmrig releases as well.

**Comment by DARK-DEVIL-66** [comment]
i compired to normal xmrig and xm-proxy also work well same as silent miner but the issue on pool side slow to update statics 

every test has maded in laptop same hashrate also

but one issue was occure in silent miner comparised with normal xm-miner (freezing laptop--> reason Using Much of Ram usage---> if enabled rootkit option i tested root kit with normal xm-miner)

**Comment by DARK-DEVIL-66** [comment]
why you can't  UnamDownloader  with indigrate our web penal it's easy for (incase user will remove the miner) we can execute again and like a loader

**Comment by UnamSanctam** [comment]
>i compired to normal xmrig and xm-proxy also work well same as silent miner but the issue on pool side slow to update statics
every test has maded in laptop same hashrate also
but one issue was occure in silent miner comparised with normal xm-miner (freezing laptop--> reason Using Much of Ram usage---> if enabled rootkit option i tested root kit with normal xm-miner)

Do you mean that your problem only happens with the rootkit enabled?

>why you can't UnamDownloader with indigrate our web penal it's easy for (incase user will remove the miner) we can execute again and like a loader

Because that's not what the downloader does, it's not a RAT.

**Comment by DARK-DEVIL-66** [comment]
> > i compired to normal xmrig and xm-proxy also work well same as silent miner but the issue on pool side slow to update statics
> > every test has maded in laptop same hashrate also
> > but one issue was occure in silent miner comparised with normal xm-miner (freezing laptop--> reason Using Much of Ram usage---> if enabled rootkit option i tested root kit with normal xm-miner)
> 
> Do you mean that your problem only happens with the rootkit enabled?
> 
> > why you can't UnamDownloader with indigrate our web penal it's easy for (incase user will remove the miner) we can execute again and like a loader
> 
maybe but it's nothing issues on pc it's happen only laptop 
whats minimem require algo random x ?


> Because that's not what the downloader does, it's not a RAT
sure ! 
just take look that 
https://www.blackhatrussia.com/virus-bot-trojan/1602-ch-miner-40-builder-panel.html

**Comment by DARK-DEVIL-66** [comment]
also https://www.blackhatrussia.com/virus-bot-trojan/1549-supreme-studio-v10-loader-miner.html

**Comment by DARK-DEVIL-66** [comment]
why decrese hashrate using xm-proxy compiring to direct pool mining

**Comment by UnamSanctam** [comment]
When you compare over something like 1 hour or 6 hours?

**Comment by DARK-DEVIL-66** [comment]
1

**Comment by DARK-DEVIL-66** [comment]
but i used normal xm-miner  
next i try to test silent miner
but i think issue is we can't set custom diff

**Comment by DARK-DEVIL-66** [comment]
i noticed powershell running at the satrting in client pc 

client pc have silent miner

**Comment by DARK-DEVIL-66** [comment]
wich coin for require low end vram for gpu mining -some pc have low VRAM status bro ?

**Comment by UnamSanctam** [comment]
>but i used normal xm-miner
next i try to test silent miner
but i think issue is we can't set custom diff

Not setting the difficulty at all is usually fine, a low hashrate miner will usually show "Offline" between shares (logically they will do so).

>i noticed powershell running at the satrting in client pc
client pc have silent miner

The silent miner won't show any powershell windows on start if that's what you mean.

>wich coin for require low end vram for gpu mining -some pc have low VRAM status bro ?

You can search up coins you can find here: https://whattomine.com/, Zeno for example is around 2GB I think can't fully remember.

**Comment by DARK-DEVIL-66** [comment]
> > but i used normal xm-miner
> > next i try to test silent miner
> > but i think issue is we can't set custom diff
> 
> Not setting the difficulty at all is usually fine, a low hashrate miner will usually show "Offline" between shares (logically they will do so).
> 
> > i noticed powershell running at the satrting in client pc
> > client pc have silent miner
> 
> The silent miner won't show any powershell windows on start if that's what you mean.
> 
> > wich coin for require low end vram for gpu mining -some pc have low VRAM status bro ?
> 
> You can search up coins you can find here: https://whattomine.com/, Zano for example is around 2GB I think, I can't fully remember.



ok  thank's  bro

**Comment by DARK-DEVIL-66** [comment]
i have one one funny qt ?

**Comment by DARK-DEVIL-66** [comment]
if you can't tell the miner count just tell how much earn month from cpu mining ?

**Comment by UnamSanctam** [comment]
I don't really spread any miners, I only code the miner and the help people with what things they ask, and in that process I do help people with a lot of miners (sometimes tens of thousands of miners) in their web panel or pool. I'm not really much of a social engineer, the miner is only a hobby for me, I get my money from my real life work or any donations that kind people donate to the project.

**Comment by DARK-DEVIL-66** [comment]
you are excellent bro ! are you sweeden

**Comment by UnamSanctam** [comment]
Thank you, yes I'm Swedish.
